doc/tutorial/getting-started/scalablytyped.md

```diff
@@ -0,0 +1,274 @@
+{% endhighlight %}
+
+We create a Laminar `canvas()` element.
+We give it a `width` and `height` using Laminar's `:=`, as we did before.
+
+For its actual content, we want Chart.js to take over.
+For that, we have to create an instance of `Chart` referencing the DOM `HTMLCanvasElement`.
+In order to bridge the world of Laminar `Element`s and Chart.js, we use `onMountUnmountCallback`.
+
+That function takes one callback executed when the element is attached to a DOM tree, and one when it is removed.
+When the element is mounted, we want to create the instance of `Chart`.
+When it is unmounted, we want to call the `destroy()` method of Chart.js to release its resources.
+
+The `mount` callback receives a `nodeCtx`, which, among other things, gives us a handle to the underlying `HTMLCanvasElement`.
+We name it `domCanvas`, and use it together with the `chartConfig` defined above to create an instance of Chart.js' `Chart` class:
+
+{% highlight scala %}
+val domCanvas: dom.HTMLCanvasElement = nodeCtx.thisNode.ref
+val chart = Chart.apply.newInstance2(domCanvas, chartConfig)
+{% endhighlight %}
+
+We store the resulting `chart` instance in a local `var optChart: Option[Chart]`.
+We will use it later to update the `chart`'s imperative data model when our FRP `dataSignal` changes.
+
+In order to achieve that, we use a `dataSignal -->` binder.
+Once the `canvas` gets mounted, every time the value of `dataSignal` changes, the callback is executed.
+
+{% highlight scala %}
+dataSignal --> { data =>
+  for (chart <- optChart) {
+    chart.data.labels = data.map(_.label).toJSArray
+    chart.data.datasets.get(0).data = data.map(_.price).toJSArray
+    chart.data.datasets.get(1).data = data.map(_.fullPrice).toJSArray
+    chart.update()
+  }
+},
+{% endhighlight %}
+
+In the callback, we get access to the `chart: Chart` instance and update its data model.
+This `-->` binder allows to bridge the FRP world of `dataSignal` with the imperative world of Chart.js.
+
+Note that we put the `-->` binder as an argument to the Laminar `canvas` element.
+This may seem suspicious, as neither `dataSignal` nor the callback have any direct relationship to the DOM canvas element.
+We do this to tie the lifetime of the binder to the lifetime of the `canvas` element.
+When the latter gets unmounted, we release the binder connection, possibly allowing resources to be reclaimed.
+
+In general, every binder must be *owned* by a Laminar element.
+It only gets *activated* when that element is mounted.
+This prevents memory leaks.
+
+Our application now properly renders the data model as a chart.
+When we add or remove data items, the chart is automatically updated, thanks to the connection established by the `dataSignal -->` binder.
+
+## Conclusion
+
+That concludes our tutorial on ScalablyTyped.
+
+Finally, we saw how to configure ScalablyTyped to get static types for external JavaScript libraries, and how to integrate a third-party component into a Laminar model.
```

Comment:
```suggestion
We saw how to configure ScalablyTyped to get static types for external JavaScript libraries, and how to integrate a third-party component into a Laminar model.
```
The "Finally" feels weird since there is a sentence before, I interpreted it as "At last, and not as "To conclude" (as I presume was intended)